test/commands/create/create.test.ts

```diff
@@ -0,0 +1,236 @@
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+//
+// SPDX-License-Identifier: Apache-2.0
+
+import { expect, test } from '@oclif/test';
+import * as fs from 'fs';
+import { packageIndexMock, velocitasConfigMock } from '../../utils/mockConfig';
+import { mockFolders, mockRestore, userHomeDir } from '../../utils/mockfs';
+import * as gitModule from 'simple-git';
+import * as exec from '../../../src/modules/exec';
+import sinon from 'sinon';
+import { simpleGitInstanceMock } from '../../helpers/simpleGit';
+import { ux } from '@oclif/core';
+import { DEFAULT_BUFFER_ENCODING } from '../../../src/modules/constants';
+const inquirer = require('inquirer');
+
+const TEST_APP_NAME = 'TestApp';
+const TEST_EXPOSED_INTERFACE_TYPE = packageIndexMock[0].exposedInterfaces[0].type;
+const TEST_EXPOSED_INTERFACE_ARG_NAME_1 = packageIndexMock[0].exposedInterfaces[0].args[0].id;
+const TEST_EXPOSED_INTERFACE_ARG_DEFAULT_1 = packageIndexMock[0].exposedInterfaces[0].args[0].default;
+const TEST_EXPOSED_INTERFACE_ARG_NAME_2 = packageIndexMock[0].exposedInterfaces[0].args[1].id;
+const TEST_EXPOSED_INTERFACE_ARG_DEFAULT_2 = packageIndexMock[0].exposedInterfaces[0].args[1].default;
+const TEST_PACKAGE_URI = packageIndexMock[0].package;
+const TEST_PACKAGE_NAME = velocitasConfigMock.packages[0].name;
+const TEST_PACKAGE_VERSION = velocitasConfigMock.packages[0].version;
+
+describe('create', () => {
+    test.do(() => {
+        mockFolders({ packageIndex: true });
+    })
+        .finally(() => {
+            mockRestore();
+        })
+        .stdout()
+        .stub(gitModule, 'simpleGit', sinon.stub().returns(simpleGitInstanceMock()))
+        .stub(exec, 'runExecSpec', () => {})
+        .stub(exec, 'awaitSpawn', () => {})
+        .command(['create', '-n', TEST_APP_NAME, '-l', 'test'])
+        .it('creates project with provided flags and generates .velocitas.json and AppManifest', (ctx) => {
+            expect(ctx.stdout).to.contain('Creating a new Velocitas project ...');
+            expect(ctx.stdout).to.contain(`... Project for Vehicle Application '${TEST_APP_NAME}' created!`);
+            expect(ctx.stdout).to.contain('Initializing Velocitas packages ...');
+            expect(ctx.stdout).to.contain(`... Downloading package: '${TEST_PACKAGE_NAME}:${TEST_PACKAGE_VERSION}'`);
+            expect(ctx.stdout).to.contain('Syncing Velocitas components!');
+
+            expect(fs.existsSync(`${process.cwd()}/.velocitas.json`)).to.be.true;
+            expect(fs.existsSync(`${process.cwd()}/app/AppManifest.json`)).to.be.true;
+
+            const velocitasConfigFile = fs.readFileSync(`${process.cwd()}/.velocitas.json`, DEFAULT_BUFFER_ENCODING);
```

Comment:
Use `PackageConfig.read()` ?

Less duplication is always better, even in tests ;-)
If we ever change PackageConfig, the tests will report a syntax issue, too.